Cognite.Simulator.Tests/UtilsTests/FileLibraryTest.cs

```diff
@@ -92,17 +92,24 @@ await modelLibTasks
                 Assert.NotNull(latest);
                 Assert.Equal(v2, latest);
 
-                var log1 = await lib.StagingArea.GetEntry(v1.Id, source.Token);
-                Assert.NotNull(log1);
-                Assert.True(log1.Parsed);
-                Assert.False(log1.Error);
-                Assert.Equal(ParsingStatus.success, log1.Status);
-
-                var log2 = await lib.StagingArea.GetEntry(v2.Id, source.Token);
-                Assert.NotNull(log2);
-                Assert.True(log2.Parsed);
-                Assert.False(log2.Error);
-                Assert.Equal(ParsingStatus.success, log2.Status);
+                var log1 = await cdf.Alpha.Simulators.RetrieveSimulatorLogsAsync(
+                    new List<Identity> { new Identity(v1.LogId) }, source.Token).ConfigureAwait(false);
+
+                var log1Data = log1.First().Data;
+                Assert.NotEmpty(log1Data);
+
+                var parsedModelEntry1 = log1Data.Where(lg => lg.Message.StartsWith("Model parsed successfully"));
+                Assert.Equal("Model parsed successfully", parsedModelEntry1.First().Message);
+                Assert.Equal("Information", parsedModelEntry1.First().Severity);
```

Comment:
is it essencial to check it twice for each or are we really only interested in the latest?